suplemon/viewer.py

```diff
@@ -96,13 +97,22 @@ def get_size(self):
         y, x = self.window.getmaxyx()
         return (x, y)
 
+    def get_scroll_pos(self):
+        return (self.y_scroll, self.x_scroll)
+
+    def get_y_scroll(self):
+        return self.y_scroll
+
+    def get_x_scroll(self):
```

Comment:
These methods `get_y_scroll` and `get_x_scroll` are redundant to directly accessing the attribute. What's their purpose?